src/pretix/control/forms/event.py

```diff
@@ -471,8 +471,8 @@ class InvoiceSettingsForm(SettingsForm):
 class MailSettingsForm(SettingsForm):
     mail_prefix = forms.CharField(
         label=_("Subject prefix"),
-        help_text=_("This will be prepended to the subject of all outgoing emails. This could be a short form of "
-                    "your event name."),
+        help_text=_("This will be prepended to the subject of all outgoing emails, formatted as [prefix]. "
```

Comment:
That's better than before, but I feel that one could still read this as "add the brackets your self". What do you think, fine-tune the help text even more ("This will be automatically formatted within brackets, e.g.") and/or add a RegexValidator that complains if you put them yourselves.